packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/SampleModuleCS.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+
+using Microsoft.ReactNative.Managed;
+
+namespace SampleLibraryCS
+{
+    // Sample ReactModule
+
+    [ReactModule]
+    internal sealed class SampleModuleCS
+    {
+        public string Name => nameof(SampleModuleCS);
```

Comment:
>public string Name => nameof(SampleModuleCS); [](start = 8, length = 45)

It must not be needed.

Reply:
The ReactNoduleAttribute by default returns name of the class.

---
In reply to: [335169016](https://github.com/microsoft/react-native-windows/pull/3426#discussion_r335169016) [](ancestors = 335169016)

Reply:
Please ignore it - I see it is being used. Maybe it should private then.

---
In reply to: [335169247](https://github.com/microsoft/react-native-windows/pull/3426#discussion_r335169247) [](ancestors = 335169247,335169016)